### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by dwillmer**

### CODE REVIEW

File: test/src/index.ts [hunk 1]
```diff
@@ -17,366 +17,501 @@ import {
...
-
-  //     });
+  });
 
-  //   });
+  describe('#keystrokes', () => {
+
+    describe('normalizeKeystrokes', () => {
+
+      it('should not register invalid keystrokes', () => {
```
Comment (sccolbert) [inline test/src/index.ts]: what is this testing? I doesn't have an `expect` clause (that I can see).

Comment (dwillmer) [inline test/src/index.ts]: That's specifically covering the edge cases in normalizeKeystrokes, but since it's not exported there's little to test directly.

Comment (sccolbert) [inline test/src/index.ts]: Then I should export it so we can test it properly. Thanks!

**Comment by sccolbert** [comment]
obviated by #17 AFAICT
